2004-11-16  Randolph Chung  <tausq@debian.org>

	* testsuite/gdb.base/unload.exp: Pass relative path to 
	gdb_get_line_number.

--- gdb-6.3/gdb/testsuite/gdb.base/unload.exp.fix2	2005-02-16 16:30:07.000000000 -0500
+++ gdb-6.3/gdb/testsuite/gdb.base/unload.exp	2005-02-16 16:30:02.000000000 -0500
@@ -1,4 +1,4 @@
-#   Copyright 2003, 2004
+#   Copyright 2003, 2004, 2005
 #   Free Software Foundation, Inc.
 
 # This program is free software; you can redistribute it and/or modify
@@ -117,7 +117,7 @@ gdb_test "info break" \
 \[0-9\]+\[\t \]+breakpoint     keep y.*PENDING.*shrfunc1.*" \
 "single pending breakpoint info"
 
-set unloadshr_line [gdb_get_line_number "unloadshr break" ${srcdir}/${subdir}/${libsrcfile}]
+set unloadshr_line [gdb_get_line_number "unloadshr break" ${libsrcfile}]
 
 gdb_test "run" \
 "Starting program.*unload.*
